gcloud/bigtable/table.py

```diff
@@ -255,3 +292,80 @@ def sample_row_keys(self):
         response_iterator = client._data_stub.SampleRowKeys(
             request_pb, client.timeout_seconds)
         return response_iterator
+
+
+def _create_row_request(table_name, row_key=None, start_key=None, end_key=None,
```

Comment:
Do you already know that this method going to be more widely used than above?  Otherwise, the signature has a lot of YAGNI.